src/data_category.js

```diff
@@ -147,73 +142,61 @@ Blockly.DataCategory.addChangeVariableBy = function (xmlList, variable) {
   //     </shadow>
   //   </value>
   // </block>
-  Blockly.DataCategory.addBlock(
-    xmlList,
-    variable,
-    "data_changevariableby",
-    "VARIABLE",
-    ["VALUE", "math_number", 1]
-  );
-};
+  addBlock(xmlList, variable, "data_changevariableby", "VARIABLE", [
+    "VALUE",
+    "math_number",
+    1,
+  ]);
+}
 
 /**
  * Construct and add a data_showVariable block to xmlList.
  * @param {!Array.<!Element>} xmlList Array of XML block elements.
  * @param {?Blockly.VariableModel} variable Variable to select in the field.
  */
-Blockly.DataCategory.addShowVariable = function (xmlList, variable) {
+function addShowVariable(xmlList, variable) {
   // <block type="data_showvariable">
   //   <value name="VARIABLE">
   //     <shadow type="data_variablemenu"></shadow>
   //   </value>
   // </block>
-  Blockly.DataCategory.addBlock(
-    xmlList,
-    variable,
-    "data_showvariable",
-    "VARIABLE"
-  );
-};
+  addBlock(xmlList, variable, "data_showvariable", "VARIABLE");
+}
 
 /**
  * Construct and add a data_hideVariable block to xmlList.
  * @param {!Array.<!Element>} xmlList Array of XML block elements.
  * @param {?Blockly.VariableModel} variable Variable to select in the field.
  */
-Blockly.DataCategory.addHideVariable = function (xmlList, variable) {
+function addHideVariable(xmlList, variable) {
   // <block type="data_hidevariable">
   //   <value name="VARIABLE">
   //     <shadow type="data_variablemenu"></shadow>
   //   </value>
   // </block>
-  Blockly.DataCategory.addBlock(
-    xmlList,
-    variable,
-    "data_hidevariable",
-    "VARIABLE"
-  );
-};
+  addBlock(xmlList, variable, "data_hidevariable", "VARIABLE");
+}
 
 /**
  * Construct and add a data_listcontents block to xmlList.
  * @param {!Array.<!Element>} xmlList Array of XML block elements.
  * @param {?Blockly.VariableModel} variable Variable to select in the field.
  */
-Blockly.DataCategory.addDataList = function (xmlList, variable) {
+function addDataList(xmlList, variable) {
   // <block id="variableId" type="data_listcontents">
   //    <field name="LIST">variablename</field>
   // </block>
-  Blockly.DataCategory.addBlock(xmlList, variable, "data_listcontents", "LIST");
+  addBlock(xmlList, variable, "data_listcontents", "LIST");
   // In the flyout, this ID must match variable ID for monitor syncing reasons
```

Comment:
Do you know what this means? Just curious.

Reply:
"monitor syncing" here I assume means displaying the value of the variable in the little monitor chip that appears on the stage if you check the box next to the block, presumably it needs the IDs to match to allow for that.